src/en/juju-systemd.md

```diff
@@ -0,0 +1,54 @@
+Title: Juju and systemd
+
+
+# Juju and systemd
+
+Below we cover the important issue of the migration of the init system used in
+Ubuntu: upstart to systemd.
+
+See [https://wiki.ubuntu.com/SystemdForUpstartUsers](https://wiki.ubuntu.com/SystemdForUpstartUsers).
+
+
+## General
+
```

Comment:
There is currently a bug where users cannot use vivid and upstart with Juju for the local provider:  https://bugs.launchpad.net/juju-core/+bug/1450092